### DISCUSSION TITLE
Oh My Posh Settings backup

### DISCUSSION THREAD

**Discussion by j0rgedev** [JanDeDobbeleer/oh-my-posh#5377] [Troubleshoot] [answered]
Hello there. I'm about to reset my PC and wouldn't like to setup my powershell again. So, is there a way to get any config files, so when restart is finished, oh my posh could take that config file and restore my powershell?

**Comment by JanDeDobbeleer** [comment, marked as answer]
@j0rgedev no, oh-my-posh can't do that. If you want to have an easy way to restore/carryover your PowerShell (or other) settings, have a look at aliae.dev.

**Reply by j0rgedev** [reply]
Thank you. I didn't know about it. I'll give it a try
